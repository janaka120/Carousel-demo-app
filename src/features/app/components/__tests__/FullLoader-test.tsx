import React from 'react';
import renderer from 'react-test-renderer';
import FullLoader from '../FullLoader';

test('FullLoader renders correctly', () => {
  const tree = renderer.create(<FullLoader />).toJSON();
  expect(tree).toMatchSnapshot();
});
