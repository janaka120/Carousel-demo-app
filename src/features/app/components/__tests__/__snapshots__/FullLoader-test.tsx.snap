// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FullLoader renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "justifyContent": "center",
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
    }
  }
>
  <ActivityIndicator
    color="#565756"
  />
  <Text
    style={
      Object {
        "color": "#565756",
        "fontSize": 29,
        "lineHeight": 39.875,
        "marginTop": 32.857142857142854,
      }
    }
  >
    Loading...
  </Text>
</View>
`;
